components/org.wso2.carbon.metrics.impl/src/main/java/org/wso2/carbon/metrics/impl/util/MetricsConstants.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2016 WSO2 Inc. (http://wso2.org)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wso2.carbon.metrics.impl.util;
+
+public class MetricsConstants {
```

Comment:
@chrishantha Constants moved to `MetricServiceImpl` on [commit:0e28587] (https://github.com/grainier/carbon-metrics/commit/0e285876fde290b95b0541c5432be7edf7e9be05)